src/terms.ml

```diff
@@ -240,6 +240,10 @@ type term =
   ; meta_value : (term, term) Bindlib.mbinder option ref
   (** Definition of the metavariable, if known. *) }
 
+(** Comparison function on metavariables. *)
+let cmp_meta m1 m2 = m1.meta_key - m2.meta_key
+let eq_meta m1 m2 = m1.meta_key = m2.meta_key
+
```

Comment:
Please, give a standard documentation comment to both functions.

Reply:
Fixed in dc44232e9d5f71ca6461469e84c26514bf9c9a32